target/linux/brcmiproc/Makefile

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (C) 2017 Yousong Zhou <yszhou4tech@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+BOARD:=brcmiproc
+BOARDNAME:=Broadcom iProc SoC Family
```

Comment:
Add SUBTARGETS:=ps225

You might find a good prototype for setting up a new target in the "samsung" target in OpenWrt.

Reply:
Modeled new commit on samsung target as much as possible